torch/utils/mkldnn.py

```diff
@@ -43,8 +43,13 @@ def __init__(self, dense_module):
         self.padding = dense_module.padding
         self.dilation = dense_module.dilation
         self.groups = dense_module.groups
-
-        self.register_buffer('weight', dense_module.weight.to_mkldnn())
+        prepacked_weight = torch._C._nn.mkldnn_reorder_conv2d_weight(
+            dense_module.weight.to_mkldnn(),
+            self.padding,
+            self.stride,
+            self.dilation,
+            self.groups)
```

Comment:
Not sure why this was not added by Facebook folks previously. Good to add it here. Let's see how FB responds.